plugins/intel-spi/intel-spi.quirk

```diff
@@ -0,0 +1,7 @@
+[PCI\VEN_8086&DEV_A150]
+Plugin = intel_spi
+Guid = INTEL_SPI_CHIPSET\SUNRISE_POINT
+
+[INTEL_SPI_CHIPSET\SUNRISE_POINT]
+IntelSpiKind = pch
+IntelSpiBar = 0xfe010000
```

Comment:
seems a bit light - is this intentional?

Reply:
keep going, the file gets massive!

Reply:
It the plugin is development only (effectively) maybe it should be a complete quirk list and just not installed by default?

Reply:
define "complete"? I'm a bit lost, sorry.

Reply:
I mean _a more complete list_ of the intel chipsets.

Reply:
Ohh! I'm missing a commit, will fixup.

Reply:
Fixed -- sorry about that.